src/OrchardCore.Modules/OrchardCore.Users/Workflows/Activities/SelectUsersInRoleTask.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Localization;
+using OrchardCore.Users.Models;
+using OrchardCore.Users.Services;
+using OrchardCore.Workflows.Abstractions.Models;
+using OrchardCore.Workflows.Activities;
+using OrchardCore.Workflows.Models;
+using OrchardCore.Workflows.Services;
+
+namespace OrchardCore.Users.Workflows.Activities
+{
+    public class SelectUsersInRoleTask : TaskActivity
+    {
+        private readonly UserManager<IUser> _userManager;
+        private readonly IUserService _userService;
+        private readonly IWorkflowExpressionEvaluator _expressionEvaluator;
+        protected readonly IStringLocalizer S;
+
+        public SelectUsersInRoleTask(UserManager<IUser> userManager, IUserService userService, IWorkflowExpressionEvaluator expressionvaluator, IStringLocalizer<SelectUsersInRoleTask> localizer)
+        {
+            _userManager = userManager;
+            _userService = userService;
+            _expressionEvaluator = expressionvaluator;
+            S = localizer;
+        }
+
+        public override string Name => nameof(SelectUsersInRoleTask);
+
+        public override LocalizedString DisplayText => S["Select Users in Role Task"];
+
+        public override LocalizedString Category => S["User"];
+
+        public WorkflowExpression<string> PropertyName
```

Comment:
maybe `Username` instead of `PropertyName`

Reply:
Will this give you access to the list of users? so someone can access the workflow input in a different task and then send them email or notify them? If so, then it's a good property to have. maybe `WorkflowOutputKeyName`?